crates/nu-command/src/database/values/sqlite.rs

```diff
@@ -428,10 +435,98 @@ pub fn open_sqlite_db(path: &Path, call_span: Span) -> Result<Connection, ShellE
 fn run_sql_query(
     conn: Connection,
     sql: &Spanned<String>,
+    params: NuSqlParams,
     ctrlc: Option<Arc<AtomicBool>>,
 ) -> Result<Value, SqliteError> {
     let stmt = conn.prepare(&sql.item)?;
-    prepared_statement_to_nu_list(stmt, sql.span, ctrlc)
+
+    prepared_statement_to_nu_list(stmt, params, sql.span, ctrlc)
+}
+
+// This is taken from to text local_into_string but tweaks it a bit so that certain formatting does not happen
+pub fn value_to_sql(value: Value) -> Result<Box<dyn rusqlite::ToSql>, ShellError> {
+    Ok(match value {
+        Value::Bool { val, .. } => Box::new(val),
+        Value::Int { val, .. } => Box::new(val),
+        Value::Float { val, .. } => Box::new(val),
+        Value::Filesize { val, .. } => Box::new(val),
+        Value::Duration { val, .. } => Box::new(val),
+        Value::Date { val, .. } => Box::new(val),
+        Value::String { val, .. } => {
+            // don't store ansi escape sequences in the database
+            // escape single quotes
+            Box::new(nu_utils::strip_ansi_unlikely(&val).into_owned())
+        }
+        Value::Binary { val, .. } => Box::new(val),
```

Comment:
I think the only addition that is missing here, which I believe was in your original, is `Value::Nothing`. I think it would be good to do something like produce and empty string or something similar that can be stored instead of just producing an error.

Reply:
Yes, the error message itself also says that the "nothing" type is a valid input. I'll go ahead and add it back. I don't think an empty string is valid, but Options implement ToSql, so I'll make it insert a `None::<String>`, which will just be null in the database

Reply:
ok, sounds good. as long as it's handled and stored in a reasonable way in the db.